src/App.svelte

```diff
@@ -2,12 +2,24 @@
   import { t } from 'svelte-i18n';
   import Header from './components/Header.svelte';
   import Footer from './components/Footer.svelte';
+  import About from './components/About.svelte';
   import { isDarkTheme } from './stores';
 </script>
 
-<main
-  class="h-screen flex flex-col justify-between {$isDarkTheme ? 'bg-dark-blue text-blue' : 'bg-white text-dark-blue'}">
-  <Header />
-  <h1 class="text-center uppercase text-5xl font-hairline">{$t('hello')} Los Crackitos</h1>
-  <Footer />
+<main>
+  <section
+    class="h-screen flex flex-col justify-between {$isDarkTheme ? 'bg-dark-blue text-blue' : 'bg-white text-dark-blue'}">
```

Comment:
You can simmplify the conditional classes and remove 3 lines if you put the `{isDarkTheme ? ...` inside the main tag class